salt/kallithea-demo/configs/production.ini

```diff
@@ -0,0 +1,604 @@
+## you must disable beaker.session.secret to use this
+#beaker.session.encrypt_key = <key_for_encryption>
+#beaker.session.validate_key = <validation_key>
+
+## sets session as invalid if it haven't been accessed for given amount of time
+beaker.session.timeout = 2592000
+beaker.session.httponly = true
+#beaker.session.cookie_path = /<your-prefix>
+
+## uncomment for https secure cookie
+beaker.session.secure = false
+
+## auto save the session to not to use .save()
+beaker.session.auto = False
+
+## default cookie expiration time in seconds `true` expire at browser close ##
+#beaker.session.cookie_expires = 3600
+
+
+############################
+## ERROR HANDLING SYSTEMS ##
+############################
+
+####################
+### [errormator] ###
+####################
+
+## Errormator is tailored to work with Kallithea, see
+## http://errormator.com for details how to obtain an account
+## you must install python package `errormator_client` to make it work
+
+## errormator enabled
+errormator = false
+
+errormator.server_url = https://api.errormator.com
+errormator.api_key = YOUR_API_KEY
+
+## TWEAK AMOUNT OF INFO SENT HERE
+
+## enables 404 error logging (default False)
+errormator.report_404 = false
+
+## time in seconds after request is considered being slow (default 1)
+errormator.slow_request_time = 1
+
+## record slow requests in application
+## (needs to be enabled for slow datastore recording and time tracking)
+errormator.slow_requests = true
+
+## enable hooking to application loggers
+# errormator.logging = true
+
+## minimum log level for log capture
+# errormator.logging.level = WARNING
+
+## send logs only from erroneous/slow requests
+## (saves API quota for intensive logging)
+errormator.logging_on_error = false
+
+## list of additonal keywords that should be grabbed from environ object
+## can be string with comma separated list of words in lowercase
+## (by default client will always send following info:
+## 'REMOTE_USER', 'REMOTE_ADDR', 'SERVER_NAME', 'CONTENT_TYPE' + all keys that
+## start with HTTP* this list be extended with additional keywords here
+errormator.environ_keys_whitelist =
+
+
+## list of keywords that should be blanked from request object
+## can be string with comma separated list of words in lowercase
+## (by default client will always blank keys that contain following words
+## 'password', 'passwd', 'pwd', 'auth_tkt', 'secret', 'csrf'
+## this list be extended with additional keywords set here
+errormator.request_keys_blacklist =
+
+
+## list of namespaces that should be ignores when gathering log entries
+## can be string with comma separated list of namespaces
+## (by default the client ignores own entries: errormator_client.client)
+errormator.log_namespace_blacklist =
+
+
+################
+### [sentry] ###
+################
+
+## sentry is a alternative open source error aggregator
+## you must install python packages `sentry` and `raven` to enable
+
+sentry.dsn = YOUR_DNS
```

Comment:
Should I add section similar to this:
https://github.com/python/psf-salt/pull/25/files#diff-45eb43e3c3a3aa644979437ed9a9d4b1R2
?